www/package.json

```diff
@@ -79,6 +80,7 @@
     "@babel/core": "^7.2.2",
     "@mdx-js/mdx": "^0.16.6",
     "@mdx-js/tag": "^0.16.6",
+    "@reach/router": "^1.2.1",
```

Comment:
Gatsby uses [`@reach/router`](https://github.com/reach/router) under the hood so we can tap into all of its features if need be.